src/files.js

```diff
@@ -45,7 +52,44 @@ const getModifiedFiles = async (octokit, repo, owner, pullNumber) => {
     return files.data.map((file) => file.filename);
 };
 
+/**
+ * Returns the content of a file
+ * @param {Object} params
+ * @param {function} params.octokit
+ * @param {string} params.repo
+ * @param {string} params.owner
+ * @param {string} params.path
+ * @param {string} [params.ref] - Default: the repository’s default
```

Comment:
never seen such way to define input interface, would it actually work with TS?

Reply:
What's the recommended way?

Reply:
same question as Bogdan, would it really work?
```
/**
 * @typdef {Object} params
 * @param {function} octokit
.....
```

Reply:
create type definition like @nitzanashi suggested, and pass it to `@param`

Reply:
@dobladov you can adopt eslint-config with JSDoc plugin it can help you validate type definitions

Reply:
So it does not come by importing our zattoo eslint config?

https://github.com/zattoo/changelog/blob/0ce90f93d0f63ca60bf4732e650fae1a377a93de/.eslintrc.js#L7

I have JSDoc validation but I'm not sure if is coming from my editor.

Reply:
it's pluggable, use

```json
"extends": [
  "@zattoo",
  "@zattoo/eslint-config/rules/jsdoc"
]
```

to make it work, I'll update documentation to make it clear

Reply:
https://github.com/zattoo/eslint-config/tree/readme#extended-with-plugins